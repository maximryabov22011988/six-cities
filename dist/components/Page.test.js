'use strict';
Object.defineProperty(exports, '__esModule', { value: true });
const React = require('react');
const renderer = require('react-test-renderer');
const Page_1 = require('./Page');
it('Page correctly render', () => {
  const tree = renderer.create(React.createElement(Page_1.default, null, null)).toJSON();
  expect(tree).toMatchSnapshot();
});
//# sourceMappingURL=Page.test.js.map
