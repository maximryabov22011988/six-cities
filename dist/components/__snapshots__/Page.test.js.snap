// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page correctly render 1`] = `
<div
  className=""
/>
`;
